app/src/main/res/menu/credit_card_editor.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <item
+        android:id="@+id/delete_credit_card_button"
+        android:icon="@drawable/ic_delete"
+        android:title="@string/credit_cards_delete_card"
+        android:visible="false"
```

Comment:
I see this option not used.
Should it have the same functionality as the "cancel" button from the fragment?
Or if not used this can be removed from the current patch and added when  there's a need for it?